src/main/deltachat.js

```diff
@@ -452,8 +452,53 @@ function snakeCaseKeys (obj) {
     send_user: obj.sendUser,
     send_pw: obj.sendPw,
     send_server: obj.sendServer,
-    send_port: obj.sendPort
+    send_port: obj.sendPort,
+    server_flags: translateSecurityToServerFlags(obj)
   }
 }
 
+function translateSecurityToServerFlags ({ mailSecurity, sendSecurity }) {
```

Comment:
Another argument for not making this into a class method is that `standard` would have caught this not being used.